spec/rspec/core/did_you_mean_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'tmpdir'
+require 'rspec/support/spec/in_sub_process'
+
+module RSpec
+  module Core
+    RSpec.describe DidYouMean do
+      if defined?(::DidYouMean::SpellChecker)
+        describe 'Call to DidYouMean' do
+          describe 'Success' do
+            let(:name) { './spec/rspec/core/did_you_mean_spec.rb' }
+            it 'should return a useful suggestion' do
```

Comment:
We prefer positive text around expectations, eg. `it 'returns a useful suggestion'`

Reply:
done